### DISCUSSION TITLE
targetFound event: ID/name of the detected image target

### DISCUSSION THREAD

**Discussion by stefanogm** [hiukim/mind-ar-js#337] [Q&A]
Hi, in a multi-targets case, how do I get the image target ID/name detected in the targetfound event (https://hiukim.github.io/mind-ar-js-doc/examples/events-handling#targetfound-and-targetlost)?
For example, I would like to know if has been detected target image 1 or 2 and then pass its value to a PHP page or something.
Thank you!
@hiukim

**Comment by hiukim** [comment]
The listener is attached to a specific target already, so you should know.  In the other words, you need to create multiple listeners for multiple targets. so you should know which one.
